modules/lib.lua

```diff
@@ -356,19 +365,31 @@ else
     ---@class LibDrawText3DParams : LibDrawTextParams
     ---@field coords vector3
     ---@field disableDrawRect? boolean
+    ---@field scale? vector2 default: `vec2(0.35,0.35)`
```

Comment:
Can you add back integer as well? I think we'll support both going forward rather than deprecate the original integer param.

Reply:
The way it's done at the moment it supports a integer as well. It just turns it into a vector2 under the hood since it make the code a bit easier to read. 

Would you like me to do it like this?
```lua
---@field scale? integer | vector2 default: `0.35`
```

Since the code already handles integers, I don't see a need to change any of the code in the actual function. Correct me if I'm wrong.

Reply:
You are correct.

Reply:
@Manason the latest commit should fix it. Can you check to confirm it's made how you wanted?